controllers/envoy/envoyconfig_controller.go

```diff
@@ -97,14 +82,14 @@ func (r *EnvoyConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 	}
 
 	// Update the ConfigRevisions list in the status
-	if err := r.consolidateRevisionList(ctx, ec, desiredVersion); err != nil {
+	if err := r.reconcileRevisionList(ctx, ec, desiredVersion); err != nil {
 		return ctrl.Result{}, err
 	}
 
 	// determine the version that should be published
 	version, err := r.getVersionToPublish(ctx, ec)
 	if err != nil {
-		if err.(cacheError).ErrorType == AllRevisionsTaintedError {
+		if err.(ControllerError).ErrorType == AllRevisionsTaintedError {
```

Comment:
cacheError is a name left from a very early version of marin3r. Has been renamed to the more generic `ControllerError` and made public to allow for type assertions from other packages, in case it is required in the future.